agot-bg-game-server/src/client/game-state-panel/ChooseHouseCardComponent.tsx

```diff
@@ -51,6 +69,6 @@ export default class ChooseHouseCardComponent extends Component<GameStateCompone
             return [];
         }
 
-        return this.props.gameState.getChoosableCards(this.props.gameClient.authenticatedPlayer.house);
+        return this.props.gameState.getChoosableCards(this.props.gameClient.authenticatedPlayer.house).sort((a, b) => a.combatStrength - b.combatStrength);
```

Comment:
Not needed. The cards are ordered in `baseGameData.json`.

Reply:
@Longwelwind Without that it starts with the strongest card going down to the 0 card which is unusual. Most users sort their cards ascending.

Reply:
Nevermind, it's needed since the cards are ordered in a map and not in an array.

Ok then!